appsec/src/helper/std_logging.hpp

```diff
@@ -100,7 +100,7 @@
 // NOLINTNEXTLINE
 #define DD_STDLOG_AFTER_WAF                                                    \
     spdlog::level::debug,                                                      \
-        "DDAS-0011-00: AppSec In-App WAF returned: {}. Took {} ms"
+        "DDAS-0011-00: AppSec In-App WAF returned({}): {} - {}. Took {} ms"
```

Comment:
This is a standard log `DDAS-XXXX-XX` and it has an expected format, so you might break some system tests by doing this... maybe it's best to separate it into its own log statement.